### PR TITLE
SRA Toolkit binary package

### PR DESCRIPTION
**Pull request by matteoguglielmi**
SRA Toolkit external package

### CODE REVIEW

**Comment by matteoguglielmi** [comment]
sra-tools external package

File: var/spack/repos/builtin/packages/sra-tools/package.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+#
+from spack import *
+
+
+class SraTools(Package):
+    """The SRA Toolkit and SDK from NCBI is a collection of tools and libraries
+       for using data in the INSDC Sequence Read Archives."""
+
+    homepage = "https://ncbi.github.io/sra-tools/"
+    url      = "http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/current/sratoolkit.current-centos_linux64.tar.gz"
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/sra-tools/package.py:48]: There's no point in using a package manager that can build packages from source, if we use generic pre-built binaries :wink:  Github repository [here](https://github.com/ncbi/sra-tools)

File: var/spack/repos/builtin/packages/sra-tools/package.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/sra-tools/package.py:25]: Please remove comment from here below, before resubmitting

**Review by alalazo — CHANGES_REQUESTED** [review]
The sources are available. Why do you use generic pre-built binaries instead?

**Pull request closed without merging**